### PR TITLE
refactor(trace-of-tab): return timestamps in microseconds

### LINKED ISSUE #2116: .timestamps should always be in microseconds
We `x / 1000` in a number of places on these timestamp values.  It's a little odd. I asked patrick if he had a guess why and he shared this punctuation-free explanation:

> something (fmp maybe?) was exposed as ts / 1000 and as the metrics got collected they're all used that way but it's now this hideous monster of misunderstand why they're not the real timestamps

We should keep these integers in their original value.
We'll break pwmetrics with this change, but that's okay. :)

This is low-priority relative to everything else but we should include it in 2.0

### PR DESCRIPTION
**Pull request by patrickhulce**
fixes #2116

### CODE REVIEW

**Comment by brendankenny** [comment]
meta comment: where should we document this? It's good to know (and important to know when you do anything with these :)

Probably trace-of-tab, but maybe also in https://github.com/GoogleChrome/lighthouse/blob/master/docs/architecture.md?

File: lighthouse-core/gather/computed/trace-of-tab.js [hunk 1]
```diff
@@ -107,8 +107,8 @@ class TraceOfTab extends ComputedArtifact {
     const timestamps = {};
 
     Object.keys(metrics).forEach(metric => {
-      timestamps[metric] = metrics[metric] && metrics[metric].ts / 1000;
-      timings[metric] = timestamps[metric] - timestamps.navigationStart;
+      timestamps[metric] = metrics[metric] && metrics[metric].ts;
+      timings[metric] = (timestamps[metric] - navigationStart.ts) / 1000;
```
Comment (brendankenny) [inline lighthouse-core/gather/computed/trace-of-tab.js]: I do wish we had a better name for these than `timings`, but may be too late :)

File: lighthouse-core/audits/consistently-interactive.js [hunk 2]
```diff
@@ -54,7 +54,7 @@ class ConsistentlyInteractiveMetric extends Audit {
    * @return {!Array<!TimePeriod>}
    */
   static _findNetworkQuietPeriods(networkRecords, traceOfTab) {
-    const traceEnd = traceOfTab.timestamps.traceEnd;
+    const traceEndTsInMs = traceOfTab.timestamps.traceEnd / 1000;
```
Comment (brendankenny) [inline lighthouse-core/audits/consistently-interactive.js]: nit: move this down to where it's used?

File: lighthouse-core/gather/computed/trace-of-tab.js (hunk 1 above)
Comment (patrickhulce) [inline lighthouse-core/gather/computed/trace-of-tab.js]: ha a battle for another day :)

File: lighthouse-core/audits/consistently-interactive.js (hunk 2 above)
Comment (patrickhulce) [inline lighthouse-core/audits/consistently-interactive.js]: aw I liked that each function in this file got this aliasing out of the way at the beginning :( 
did it make you look for its use too much right afterwards?

Comment (brendankenny) [inline lighthouse-core/audits/consistently-interactive.js]: ha, yeah, exactly. I figured that's what this was doing, but then my next thought was, "wait, what is this even used for?"

**Comment by patrickhulce** [comment]
got another PR comin' with some more groovy docs 📜  📝

**Pull request merged**